lib/formatters.ts

```
@@ -11,32 +11,95 @@ export interface FormatterConfig {
+  switch (language) {
+    case 'it':
+    case 'it-IT':
+      return {
+        ...baseConfig,
+        locale: 'it-IT',
+        timezone: 'Europe/Rome',
+        currency: 'EUR',
+        use24Hour: true,
+        startOfWeek: 1 // Monday
+      }
+    case 'en':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)